### PR TITLE
import Observable and of from rxjs separate file locations

### PR DESCRIPTION
**Pull request by hemanthguptha**
In hero.service.ts file we need to change import { Observable, of } from 'rxjs'; into import { Observable } from 'rxjs';
and import { of } from 'rxjs/observable/of';

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/content/tutorial/toh-pt4.md [hunk 1]
```diff
@@ -195,6 +195,8 @@ In a [later tutorial on HTTP](tutorial/toh-pt6), you'll learn that Angular's `Ht
 In this tutorial, you'll simulate getting data from the server with the RxJS `of()` function.
 
 Open the `HeroService` file and import the `Observable` and `of` symbols from RxJS.
+In 'HeroService' file we need to import { Observable, of } from 'rxjs' separately as import { Observable } from 'rxjs';
```
Comment (alfaproject) [inline aio/content/tutorial/toh-pt4.md]: No, we don't.

**Comment by jotatoledo** [comment]
https://www.academind.com/learn/javascript/rxjs-6-what-changed/#import-statement-update-path

https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/migration.md#import-paths

**Comment by hemanthguptha** [comment]
Thank you so much Sir for giving this updates.

**Comment by brandonroberts** [comment]
Closing as not needed.

**Pull request closed without merging**